conans/test/integration/generators/json_test.py

```diff
@@ -26,7 +29,8 @@ def package_info(self):
         client.run("install Hello/0.1@lasote/testing -g json")
         conan_json = client.load("conanbuildinfo.json")
         data = json.loads(conan_json)
-
+        self.assertEqual(data["dependencies"][0]["version"], "0.1")
+        self.assertIsNone(data["dependencies"][0]["description"])
```

Comment:
Description keep being None even if defined?

Reply:
Accessing the description is even more hacky, the ConanfileInterface doesn't contain that field (and I'm not sure why it should/and not other) that's why I left it (doesn't feel so critical as the version field)

Reply:
I've tested this fix and it works for the version field, description still null but that is minor. Thanks for the quick fix